test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -54,9 +62,24 @@ var _ = Describe("Google Cloud Filestore CSI Driver", func() {
 			// Delete Disk
 			client.DeleteVolume(vol.GetVolumeId())
 			Expect(err).To(BeNil(), "DeleteVolume failed")
+
+			proj, zone, _ := instance.GetIdentity()
+			instanceUri := fmt.Sprintf(instanceURIFormat, proj, zone, volName)
+			_, err := fileInstancesService.Get(instanceUri).Do()
+			Expect(err).NotTo(BeNil(), "Could get deleted disk from cloud directly")
 		}()
 
-		// TODO validate the Filestore instance creation at the cloud provider layer
+		// Validate Disk Created
+		proj, zone, _ := instance.GetIdentity()
+		instanceUri := fmt.Sprintf(instanceURIFormat, proj, zone, volName)
+		inst, err := fileInstancesService.Get(instanceUri).Do()
```

Comment:
Let's create a helper function to get the instance.

Reply:
Done.